src/types.db

```diff
@@ -192,6 +192,27 @@ snr			value:GAUGE:0:U
 spam_check		value:GAUGE:0:U
 spam_score		value:GAUGE:U:U
 spl			value:GAUGE:U:U
+stenographer_aged_files value:COUNTER:0:U 
+stenographer_current_files value:COUNTER:0:U
+stenographer_http_request_query_POST_bytes value:COUNTER:0:U
+stenographer_http_request_query_POST_completed value:COUNTER:0:U
+stenographer_http_request_query_POST_nanos value:COUNTER:0:U
+stenographer_index_base_lookup_nanos value:COUNTER:0:U
+stenographer_index_base_lookups_finished value:COUNTER:0:U
+stenographer_index_base_lookups_started value:COUNTER:0:U
+stenographer_index_set_lookup_nanos value:COUNTER:0:U
+stenographer_index_set_lookups_finished value:COUNTER:0:U
+stenographer_index_set_lookups_started value:COUNTER:0:U
+stenographer_indexfile_current_reads value:COUNTER:0:U
```

Comment:
If I read `indexfile.go` correctly, this is a *gauge* value:

```go
indexCurrentReads.IncrementBy(-1)
```

I would use the `pending_operations` type for this.